dotCMS/src/main/java/com/dotmarketing/business/UserAPI.java

```diff
@@ -91,6 +91,8 @@
 	 */
 	public List<User> getUsersByName(String filter, int start,int limit, User user, boolean respectFrontEndRoles) throws  DotDataException;
 
+	public long getCountUsersByName(String filter) throws DotDataException;

```

Comment:
I think the name should be "getUserCountByName" instead of "getCountUsersByName". Same thing for all other similar methods in other classes.